src/components/topBar/tests/TopBar.test.tsx

```diff
@@ -24,12 +24,14 @@ const apps: Metadata[] = [
         url: '/app1',
         appColor: 'blue',
         hiddenInAppsMenu: false,
+        hiddenUserInformation: false,
     },
     {
         name: 'App2',
         url: '/app2',
         appColor: 'green',
         hiddenInAppsMenu: true,
+        hiddenUserInformation: false,
```

Comment:
could set one with true to test both case no ?

Reply:
Done